mappings/net/minecraft/util/math/Quaternion.mapping

```diff
@@ -0,0 +1,31 @@
+CLASS net/minecraft/class_1158 net/minecraft/util/math/Quaternion
+	FIELD field_5656 components [F
+	METHOD <init> (FFFF)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
+	METHOD <init> (FFFZ)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
+	METHOD <init> (Lnet/minecraft/class_1158;)V
+		ARG 1 other
+	METHOD <init> (Lnet/minecraft/class_1160;FZ)V
+		ARG 1 direction
+		ARG 2 magnitude
+		ARG 3 normalize
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 o
+	METHOD method_16002 fSin (F)F
+		ARG 0 value
+	METHOD method_16003 fCos (F)F
+		ARG 0 value
+	METHOD method_4921 x ()F
+	METHOD method_4922 y ()F
+	METHOD method_4923 z ()F
+	METHOD method_4924 w ()F
```

Comment:
These should be `getX`, etc. to respect the getter conventions (look at other vector classes, they use `getX`, etc.).

Reply:
When I looked the other vector classes weren't using `getX()`. It was just `x()`.

Reply:
net/minecraft/client/util/math/Vector3f

    public float x() {
        return this.components[0];
    }
    
    public float y() {
        return this.components[1];
    }
    
    public float z() {
        return this.components[2];
    }

Reply:
Not sure which vector class you looked at, but all the methods I've seen that return coordinates are named `getX`, `getY`, and `getZ`. If there's some other getter named `x` rather than `getX`, that one is wrong.

Reply:
Not using the `getX` naming will also make it hard for kotlin developers

Reply:
https://github.com/FabricMC/yarn/issues/789

Reply:
Coolio

Reply:
@killjoy1221 How does the naming make things harder for Kotlin devs? It's just a method name, brah.